cis194/week2/zehua/test/ExercisesSpec.hs

```diff
@@ -1,9 +1,16 @@
 module ExercisesSpec where
 
+import Data.List (sort)
 import Test.Hspec
 import Exercises
 import Log
 
+-- ad-hoc instance def to make it sortable
+instance Ord LogMessage where
```

Comment:
probably not the right thing to do here 😓 just trying to be lazy in the test to not to have to manually sort the results and to assert that `inOrder . build === sort` (not exactly, need a filter to remove unknowns before `sort`).

Reply:
not sure how to declare this only for the test for `ex4`, as it's only used in https://github.com/fp-works/2019-winter-Haskell-school/pull/31/files#diff-70139cd9b5806c5cd766b5ccfea9affcR125

Reply:
followed the advice from the compiler to use `newtype` to do this.

Ideally, still need a way to make it visible only to the test for `ex4`.